manual/src/cmds/intf-c.etex

```diff
@@ -2522,16 +2529,16 @@ after:
 \end{verbatim}
 
 Note: the "acquire" and "release" functions described above were
-introduced in OCaml 3.12.  Older code uses the following historical
-names, declared in "<caml/signals.h>":
+introduced in OCaml 3.12. The following equivalent historical names
+are declared in "<caml/signals.h>" and are sometimes used instead:
```

Comment:
I don't think this change is necessary - apart from anything else, it suggests that `caml_enter_blocking_section` is a suitable alias as opposed to an older one (although I confess I still use it...)

Reply:
The previous text is outdated. `caml_enter/leave_blocking_section` are more used than the alternative and are now better-documented. They have the `no_pending` variant, which was not added with a "release" alias. So it needs to be changed so that it does not make it sound like most of the code on opam uses a bad/obsolete alias.